playbooks/roles/caasp4/templates/inventory-caasp4.yml.j2

```diff
@@ -2,7 +2,7 @@
 caasp-masters:
   hosts:
 {% for master_ip in _terraform_json_output.ip_masters.value %}
-    caasp-master-{{ loop.index0 }}:
+    master-{{ loop.index0 }}:
```

Comment:
Could describe (in the commit message) what the problem with `caasp-master-X` is? Where is that naming used afterwards? In Airship?

Reply:
kubectl will report those names as said on the commit msg. It just makes sense to keep the same naming convention that we had before + what kubectl reports as its used afterwards for labeling the nodes for example.

I will try to expand the commit msg to make it more clear.

Reply:
Hm, but doesn't caasp3 deployment actually use the hostnames of the nodes? (I admit `caasp-master-X` etc. are not hostnames though)

Reply:
it uses the inventory names, which in caasp3 are master-X and worker-X
Changing them in the caasp4 inventory was a mistake.

Im not sure why is this such a big deal tbh? We are just syncing the hostname with the inventory name so we can use the same playbooks as in caasp3 with caasp4. Who care what the names of the caasp workers/masters are in our inventory?

Reply:
I'm not saying it's a big deal, I only think your change does not make it same as in caasp3 (which probably does not matter)